### PR TITLE
Rename everything and remove credits

### PR DESCRIPTION
**Pull request by matt-hu**
Rename everything and comments out the credits

- [x] I made sure the code compiles on my machine.
- [x] I made sure there are no unnecessary changes in the code.
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving.
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?".
- [x] I hereby reaffirm that I am licensing my contribution under the GNU General Public License v2.0 or later (`GPL-2.0-or-later`).

### CODE REVIEW

**Comment by AnErrupTion** [comment]
The problem here is that in the commit b726932, you have replaced important strings that will prevent people from building Audacium. Aside from that, instead of removing all credits and contributors you could have replaced them with the current Audacium ones.

**Comment by BenBE** [comment]
Two major notes on this PR:

1. The first one is a more or less formal one: Avoid cross-merging branches in a PR. It's absolutely bad style and makes reviewing such a PR a nightmare. Please prefer rebasing your PR on top of the branch you want to merge it into and refresh regularly in case new commits are added in the target branch.

2. Okay, now for the elephant in the room: I don't think you can simply remove the list of people who contributed to your program without replacement. Even if you have quite good reason to somewhat hide certain names on this list. Make sure this list of contributors is replicated somewhere else (e.g. in `CONTRIBUTORS.md` or alike) as otherwise this is an unjust appropriation of the work of those people.

**Comment by AnErrupTion** [comment]
> Two major notes on this PR:
> 
> 1. The first one is a more or less formal one: Avoid cross-merging branches in a PR. It's absolutely bad style and makes reviewing such a PR a nightmare. Please prefer rebasing your PR on top of the branch you want to merge it into and refresh regularly in case new commits are added in the target branch.
> 2. Okay, now for the elephant in the room: I don't think you can simply remove the list of people who contributed to your program without replacement. Even if you have quite good reason to somewhat hide certain names on this list. Make sure this list of contributors is replicated somewhere else (e.g. in `CONTRIBUTORS.md` or alike) as otherwise this is an unjust appropriation of the work of those people.

For the second point, you could add in the "Thanks" category something like "The Audacity Team" so you don't have to put their names again and it would be efficient.

**Comment by matt-hu** [comment]
Ok thanks for the feedback. I did check that it built on my PC before I submitted, but I'll double check that.

I'm not going to have time to fix the problems now unfortunately, I'm really sorry.

**Comment by nulldg** [comment]
https://github.com/SartoxOnlyGNU/audacium/pull/17/commits/b7269320e4063f7c893cf4be7eddf752f7b77207 looks like an automated replacement. I strongly recommend against doing this. Audacity has far too much technical debt to be doing stuff like this without reviewing each modified line, especially when the changes are internal and don't benefit the end-user.
Additionally, it breaks compatibility with old installs of audacity, creating duplicate settings and environmental variabbles.
A lot of people switching to this fork don't want the inconvenience of having to set up their configuration again. If you really want to do this anyway, at least add code to copy/port over values from audacity (if any). I think this should be trivial to do.

File: BUILDING.md [hunk 1]
```diff
@@ -163,16 +163,16 @@ On Linux it is possible to build Audacium using (almost) only the libraries prov
...
+        -Daudacium_lib_preference=system \
+        -Daudacium_obey_system_dependencies=On \
          ../audacium
 ```
 
 There are a few cases when the local library build is preferred:
 
 1. **wxWidgets**: While Audacium on **Linux** uses vanilla version of wxWidgets, we **require** that version **3.1.3** is used. This version is not available in most of the distributions.
-2. **portaudio-v19**: Audacium currently uses [some private APIs](https://github.com/audacity/audacity/issues/871), so using system portaudio is not yet possible.
+2. **portaudio-v19**: Audacium currently uses [some private APIs](https://github.com/audacium/audacium/issues/871), so using system portaudio is not yet possible.
```
Comment (nulldg) [inline BUILDING.md:175]: Link is invalid

**Comment by AnErrupTion** [comment]
So I modified the credits myself, however I want to keep it open until Matt sees it so that he can learn of his mistakes (to not mass replace a string, especially on a project this big).

**Comment by matt-hu** [comment]
Hi everyone,

Firstly, thanks for all the feedback and I'm truly sorry for wasting all of your time. As guessed, I did use an automated tool to switch all the mentions, I thought I'd gone through and changed back all the critical mentions but there were some oversights. With hindsight, this was clearly a bad idea.

Secondly, could we have an issue or wiki page somewhere with a list of smaller tasks for people like myself to complete? I guess part of the problem was I wanted to help out but didn't know where to start. I understand if this isn't something you want/have time to do though.

**Comment by BenBE** [comment]
> Hi everyone,
> 
> Firstly, thanks for all the feedback and I'm truly sorry for wasting all of your time.

You're welcome. And don't worry about the time it took to review your PR: Everyone once started somehow and even after years of experience people still make mistakes. Spotting these is what a review should do; alongside of advising how things can be improved.

> As guessed, I did use an automated tool to switch all the mentions, I thought I'd gone through and changed back all the critical mentions but there were some oversights. With hindsight, this was clearly a bad idea.

Sometimes making a mistake is a valuable experience. ;-) But TBH: Even I sometimes do such replacements automated and check them afterwards. The problem in this case really isn't the automation, but getting enough concentration for the whole process to spot each and every mistake.

Every change you do in a project, be it your own or a contribution somewhere else, should be handled with care. And you can recognize good developer communities by the care they put into validating contributions and checking for things outside the obvious path, and helping with the things you might have not seen.

> Secondly, could we have an issue or wiki page somewhere with a list of smaller tasks for people like myself to complete? I guess part of the problem was I wanted to help out but didn't know where to start. I understand if this isn't something you want/have time to do though.

A good starting point usually is some "good first issue" label some projects use which includes things like small features or cleanup or maintenance tasks that don't change the structure of the project. Or a small wish list of things that people may want to look into that aren't exactly on the usual development path.

**Comment by matt-hu** [comment]
> A good starting point usually is some "good first issue" label some projects use which includes things like small features or cleanup or maintenance tasks that don't change the structure of the project. Or a small wish list of things that people may want to look into that aren't exactly on the usual development path.

Ok, I'll keep an eye out for things like that. I though I might have a look at documentation and distribution until then.

**Pull request closed without merging**